src/appkit/haptics.rs

```diff
@@ -0,0 +1,78 @@
+use std::convert::TryFrom;
+
+use objc::{class, msg_send, runtime::Object, sel, sel_impl};
+use objc_id::ShareId;
+
+use crate::foundation::NSUInteger;
+
+#[derive(Clone, Debug)]
+pub struct HapticFeedbackPerformer(pub ShareId<Object>);
+impl HapticFeedbackPerformer {
```

Comment:
This is so dumb-pedantic of me, but can you linebreak between the struct and impl?

Reply:
LMAO no problem